### PR TITLE
upd(github-cli-deb): `2.6.0` -> `2.7.0`

### PR DESCRIPTION
**Pull request by WRM-42**

### CODE REVIEW

File: packages/github-cli-deb/github-cli-deb.pacscript [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
+breaks="${gives}-bin ${gives}-git"
+replace="${gives}"
+version="2.7.0"
 url="https://github.com/cli/cli/releases/download/v${version}/${gives}_${version}_linux_amd64.deb"
 description="GitHub's official command line tool"
-hash="028b215bba515c3922fa63f033fbb036ce1e54d6b42bfefc808f1c6a081b2e09"
+hash="ce393ac41fd97fd27ebdaf81de882150b6d4fc4444afe8f490d74d1312260481"
 maintainer="WRM-42 <y8bsbahy@anonaddy.me>"
-repology=("project: github-cli")
+repology=("project: github-cli" "visiblename: ${gives}-bin")
```
Comment (wizard-28) [inline packages/github-cli-deb/github-cli-deb.pacscript:10]: Why is `visiblename` filter added?

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/github-cli-deb/github-cli-deb.pacscript (hunk 1 above)
Comment (WRM-42) [reply packages/github-cli-deb/github-cli-deb.pacscript:10]: It won't work without `visiblename` for some reason

Comment (wizard-28) [reply packages/github-cli-deb/github-cli-deb.pacscript:10]: It works for me. Can you report a bug on the [PacUp](https://github.com/pacstall/pacup) repository?

Comment (WRM-42) [reply packages/github-cli-deb/github-cli-deb.pacscript]: I did another test and it works now

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**